### PR TITLE
Update continuous-profiling & dashboard-profiling

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Update continuous-profiling & dashboard-profiling

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7389
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
I've finished the self-check of this PR. PTAL. Thanks.

File: dashboard/continuous-profiling.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        tiup cluster reload ${cluster-name} --role prometheus
+        ```
+
+    After reloading Prometheus, you have finished operations on the control machine.
+
+2. Enable the Continuous Profiling feature.
```
Comment (qiancai) [inline dashboard/continuous-profiling.md:106]: 这个步骤英文进行了步骤调整，请改下中文，保持中英一致。

File: dashboard/continuous-profiling.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+## Access the page
+
+You can access the instance profiling page using either of the following methods:
+
+- After logging into TiDB Dashboard, click **Advanced Debugging** > **Profile Instances** > **Continuous Profile** on the left navigation bar.
+
+    ![Access](/media/dashboard/dashboard-conprof-access.png)
+
+- Visit <http://127.0.0.1:2379/dashboard/#/continuous_profiling> via your browser your browser. Replace `127.0.0.1:2379` with the actual PD instance address and port.
```
Comment (qiancai) [inline dashboard/continuous-profiling.md:124]:
```suggestion
- Visit <http://127.0.0.1:2379/dashboard/#/continuous_profiling> via your browser. Replace `127.0.0.1:2379` with the actual PD instance address and port.
```

File: dashboard/continuous-profiling.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+## View profiling history
+
+After starting continuous profiling, you can view the profiling result on the instance profiling page.
+
+![Profiling history](/media/dashboard/dashboard-conprof-history.png)
+
+Performance profiling runs in the background, and refreshing or exiting the current page will not terminate a running performance profiling task.
```
Comment (qiancai) [inline dashboard/continuous-profiling.md:132]:
```suggestion
Performance profiling runs in the background. Refreshing or exiting the current page will not terminate a running performance profiling task.
```

File: dashboard/continuous-profiling.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+## Download profiling result
+
+On the profiling result page, you can click **Download Profiling Result** in the upper right corner to download all profiling results.
```
Comment (qiancai) [inline dashboard/continuous-profiling.md:136]:
```suggestion
On the profiling result page, you can click **Download Profiling Result** in the upper-right corner to download all profiling results.
```

File: dashboard/continuous-profiling.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+![Download profiling result](/media/dashboard/dashboard-conprof-download.png)
+
+You can also click an individual instance to view its profiling result:
+
+![View the profiling result of an instance](/media/dashboard/dashboard-conprof-single.png)
+
+## Disable continuous profiling
```
Comment (qiancai) [inline dashboard/continuous-profiling.md:144]:
```suggestion
## Disable Continuous Profiling
```

File: dashboard/dashboard-profiling.md [hunk 6]
```diff
@@ -4,17 +4,15 @@ summary: Learn the instance profiling page of TiDB Dashboard.
 aliases: ['/docs/dev/dashboard/dashboard-profiling/']
 ---
 
-# Instance Profiling Page
+# Manual Instance Profiling Page
 
-The instance profiling page summarizes the internal performance data of each TiDB instance, TiKV instance, and PD instance. With this page, you can view the instance profiling without restarting these instances.
-
-The summarized performance data can be displayed as a flame graph or a directed acyclic graph, which visually shows the internal operations and proportions performed on the instances during the performance summary period. With this graph, you can quickly learn the main CPU resource consumption of these instances.
+On the manual instance profiling page, you can collect continuous performance data of TiDB, TiKV, PD, and TiFlash instances without restarting any of them.The data collected can be displayed on a flame graph or a directed acyclic graph, which visually shows the internal operations and proportions performed on the instances during the performance summary period. With this graph, you can quickly learn the CPU resource consumption of these instances.
```
Comment (qiancai) [inline dashboard/dashboard-profiling.md:9]:
```suggestion
On the manual instance profiling page, you can collect continuous performance data of TiDB, TiKV, PD, and TiFlash instances without restarting any of them. The data collected can be displayed on a flame graph or a directed acyclic graph. The graph visually shows what internal operations are performed on the instances during the performance summary period and the corresponding proportions. With this graph, you can quickly learn the CPU resource consumption of these instances.
```

File: dashboard/continuous-profiling.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+- TiDB/PD: CPU profile, Heap, Mutex, Goroutine (debug=2)
+- TiKV: CPU profile
+
+## Enable continuous profiling
+
+To enable Continuous Profiling, you need to first check the version information and then configure related settings on the control machine and TiDB Dashboard.
+
+### Version check
+
+Before enabling this feature, check the version of the TiUP cluster and ensure that it is 1.7.0 or above. If it is earlier than 1.7.0, upgrade it first and then reload Prometheus.
```
[the remark this answers is not in the document]
Comment (shichun-0415) [reply dashboard/continuous-profiling.md:35]:
```suggestion
Before enabling this feature, check the version of the TiUP cluster and ensure that it is 1.7.0 or above. If it is earlier than 1.7.0, upgrade it first.
```

**Review by qiancai — APPROVED** [review]

File: dashboard/continuous-profiling.md [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+
+## Disable Continuous Profiling
+
+1. On TiDB Dashboard, click **Advanced Debugging** > **Profile Instances** > **Continuous Profile** on the left navigation bar. Click **Settings**.
+
+2. In the popped-up window, switch off the button under **Enable Feature**.
+
+3. In the dialog box of **Disable Continuous Profiling Feature**, click **Disable**.
+
+3. Click **Save**.
```
Comment (TomShawn) [inline dashboard/continuous-profiling.md:158]:
```suggestion
4. Click **Save**.
```

**Review by lilyjazz — APPROVED** [review]

File: dashboard/continuous-profiling.md [hunk 9]
```diff
@@ -0,0 +1,160 @@
+---
+title: TiDB Dashboard Continuous Profiling
+summary: Learn how to enable Continuous Profiling and observe system conditions by using this feature.
+aliases: ['/docs/dev/dashboard/continuous profiling/']
```
Comment (shichun-0415) [inline dashboard/continuous-profiling.md:4]:
```suggestion
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**